pypika/__init__.py

```diff
@@ -83,4 +83,4 @@
 
 __author__ = 'Timothy Heys'
 __email__ = 'theys@kayak.com'
-__version__ = '0.28.1'
+__version__ = '0.29.0'
```

Comment:
Could you please revert the version increment? Otherwise we run the risk of conflicts if something else gets merged ahead of this PR. I'll set this on the master branch once merged.

Reply:
Actually, nevermind that, I'll just merge now so no problem.